src/credentials/attestation_verifier/DeviceAttestationDelegate.h

```diff
@@ -48,14 +48,29 @@ class DeviceAttestationDelegate
     /**
      * @brief
      *   This method is invoked when device attestation fails for a device that is being commissioned. The client
```

Comment:
It can also be called on success now, no?  This should probably describe the actual semantics of when this is called (e.g. I don't obviously see it being called on attestation success if ShouldWaitAfterDeviceAttestation is false).

Reply:
I think perhaps my documentation is not clear enough. The original point of the attestation delegate is to handle the fail case, and override it with a call to Continue...

The new use case is to also call OnDeviceAttestationCompleted on success, if ShouldWaitAfterDeviceAttestation returns true.